Introspection/package_tests.py

```diff
@@ -155,9 +166,8 @@ def find_adhoc_bins_libs(self, installed_packages=[]):
             # installed by RPM
             print len(bins)
             bins = list(set(bins) - set(self.get_installed_files(package)))
-            print len(bins)
 
-        return bins
+        return [b for b in bins if not self.match_bin_with_pkg(b)]
```

Comment:
Here you want to get metadata only when you have **None** return from match_bin_with_pkg ?